nevis/_bng.py

```diff
@@ -492,3 +493,13 @@ def pub(name=None):
     'MacSorleys': Coords(gridx=258809, gridy=665079),
     'Sheffield tap': Coords(gridx=435847, gridy=387030),
 }
+
+
+def schiehallion():
+    """ Returns a tuple, the hill object of Schiehallion and the boundaries of
+    a neighborhood of it."""
```

Comment:
```suggestion
    """
    Returns a tuple ``(hill, boundaries)`` containing the :class:`Hill` object 
    for Schiehallion and boundaries ``(x_lower, x_upper, y_lower, y_upper)``
    defining its neighbourhood.
    
    This can be used as an easier test problem for local methods.
    """
```